stubs/parsimonious/parsimonious/exceptions.pyi

```diff
@@ -0,0 +1,24 @@
+from parsimonious.expressions import Expression as Expression
```

Comment:
Most of these (outside `__init__.pyi`) probably shouldn't be using `as`; that indicates that the file is re-exporting the name.